File: Codeforces/136A_Presents.py
n = int(input())
gifts = list(map(int, input().split()))
result = []

for i in range(len(gifts)):
    result.append(0)

for j in range (len(gifts)):
    result[gifts[j] - 1] = j + 1

for r in result:
    print(r, end=' ')

print()

File: Codeforces/136A_Presents.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    
    int n;
    cin >> n;

    vector <int> gifts(n);
    vector <int> result(n);

    for (int i = 0; i < n; i++)
        cin >> gifts[i];

    for (int i = 0; i < gifts.size(); i++)
        result[gifts[i] - 1] = i + 1;
    
    for (int i = 0; i < result.size(); i++)
        cout << result[i] << " ";

    cout << endl;

    return 0;
}
